spec/services/merge_requests/merge_service_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe MergeRequests::MergeService do
+  let(:user) { create(:user) }
+  let(:user2) { create(:user) }
+  let(:merge_request) { create(:merge_request, assignee: user2) }
+  let(:project) { merge_request.project }
+
+  before do
+    project.team << [user, :master]
+    project.team << [user2, :developer]
+  end
+
+  describe :execute do
+    context "valid params" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.